recipe/meta.yaml

```diff
@@ -36,22 +35,24 @@ test:
     - tableauserverclient.server
     - tableauserverclient.server.endpoint
   requires:
-    # Despite argparse being in pyproject.toml as optional for testing purposes, it's only used in
-    # the samples, not the tests.
-    # Including it makes Conda unable to resolve the environment properly, because argparse is so
-    # ancient at this point. This will likely need to be commented out/removed going forward.
-    # - argparse
-    - black
+    - pip
     - mock
-    - mypy
     - pytest >=7.0
     - requests-mock >=1.0,<2.0
+  source_files:
+    - test
+  commands:
+    - pip check
+    # ignore completely tests that depend on data files not contained in the tarball
+    - pytest test --ignore test/test_workbook.py --ignore test/test_view.py --ignore test/test_flow.py --ignore test/test_fileuploads.py --ignore test/test_filesys_helpers.py --ignore test/test_datasource.py
 
 about:
   home: https://github.com/tableau/server-client-python
-  license: MIT
-  license_family: MIT
-  license_file: LICENSE
+  license: MIT and CC0-1.0
```

Comment:
That's wrong. The license is MIT. Versioneer is not included in the file artifacts.

Reply:
ok